src/pages/resultsView/ResultsViewPageStore.ts

```diff
@@ -102,6 +102,17 @@ export class ResultsViewPageStore {
         invoke: async () => fetchSamples(this.sampleIds, this.studyId)
     }, []);
 
+    readonly studies = remoteData({
+        invoke: async()=>([await client.getStudyUsingGET({studyId: this.studyId})])
+    }, []);
+
+    @computed get studyToCancerType() {
+        return this.studies.result.reduce((map:{[studyId:string]:string}, next:CancerStudy)=>{
```

Comment:
Although this is a small function, it might still be better to move this into `StoreUtils` for testing purposes. Well, currently most of the `StoreUtils` functions are untested, but maybe one day we add more tests :)